core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/io/FastDoubleParser.kt

```diff
@@ -0,0 +1,233 @@
+package org.jetbrains.kotlinx.dataframe.impl.io
+
+import ch.randelshofer.fastdoubleparser.ConfigurableDoubleParser
+import ch.randelshofer.fastdoubleparser.NumberFormatSymbols
+import io.github.oshai.kotlinlogging.KotlinLogging
+import org.jetbrains.kotlinx.dataframe.api.ParserOptions
+import java.nio.charset.Charset
+import java.text.DecimalFormatSymbols
+import java.text.NumberFormat
+import java.text.ParsePosition
+import java.util.Locale
+
+private val logger = KotlinLogging.logger {}
+
+// (lowercase) strings that are recognized to represent infinity and NaN in doubles in all locales
+private val INFINITIES = arrayOf("∞", "inf", "infinity", "infty")
```

Comment:
How did you collect this information? Are some standards somewhere available?

Reply:
We already had "inf" and "infinity". I added "infty" because it's used in the math world, like LaTeX, and finally, I supposed the infinity symbol itself should also be recognizable. Any others we should add?